src/Data/GeneralMarketingConsent/GeneralMarketingConsentDataItem.ts

```diff
@@ -6,7 +6,7 @@ const GENERAL_TOPIC_ID = 'general'
 export const GeneralMarketingConsent = provideDataItem(
   'GeneralMarketingConsent',
   {
-    attributes: {},
+    attributes: { isConsentGiven: 'boolean' },
```

Comment:
Can you provide a nice german and english title?

Reply:
AFAICS `provideDataItem` doesn't accept an async callback for `attributes`?

Reply:
Ah. I didn't know. Can you file an SDK issue for this?